### PR TITLE
Issue 485

### LINKED ISSUE #485: "Hostname" and "webAddress" printed in punycode instead of unicode (IDN)
Hey there, 
in order to use this software with intertionalized domain names (IDN) is auspicable to print unicode instead of punycode.
 
you can see a live example here -> https://краш.мкд

### PR DESCRIPTION
**Pull request by francescayeye**
closes #485

### CODE REVIEW

File: server/handlers.go [hunk 1]
```diff
@@ -36,6 +36,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"golang.org/x/net/idna"
```
Comment (stefanbenten) [inline server/handlers.go:39]: Lets move this down into the right section.

**Review by stefanbenten — COMMENTED** [review]
Small linting/style suggestion, otherwise looks good to me.

File: server/handlers.go (hunk 1 above)
Comment (aspacca) [reply server/handlers.go:39]: hoped it was spotted by the linter :)

**Review by stefanbenten — APPROVED** [review]

File: server/handlers.go (hunk 1 above)
Comment (stefanbenten) [reply server/handlers.go]: I'll add a lint step for that 👍

**Pull request merged**